packages/peregrine/lib/talons/CheckoutPage/ShippingInformation/useShippingInformation.js

```diff
@@ -0,0 +1,79 @@
+import { useCallback, useEffect, useMemo } from 'react';
+import { useLazyQuery } from '@apollo/react-hooks';
+
+import { useAppContext } from '../../../context/app';
+import { useCartContext } from '../../../context/cart';
+import { MOCKED_ADDRESS } from '../../CartPage/PriceAdjustments/ShippingMethods/useShippingForm';
+
+export const useShippingInformation = props => {
+    const {
+        onSave,
+        queries: { getShippingInformationQuery }
+    } = props;
+
+    const [, { toggleDrawer }] = useAppContext();
+    const [{ cartId }] = useCartContext();
+
+    const [fetchShippingInformation, { called, data, loading }] = useLazyQuery(
+        getShippingInformationQuery
+    );
+
+    useEffect(() => {
+        if (cartId) {
+            fetchShippingInformation({
+                variables: {
+                    cartId
+                }
```

Comment:
Tommy fixed this - a user re-estimating on the cart page now causes this to display as the original edit form rather than the card/update view. This is because we resubmit the mock data when estimating so when we determine the initial state of shipping data and strip the mock data we don't have a `city`, which is the compared value that we are using to indicate the lack of data.